### ISSUE TITLE
support gdstk cells?

### ISSUE THREAD

**Issue by joamatab** [flaport/flayout#3]
How can we support gdsfactory cells from version 6.0.0 or above?

![image](https://user-images.githubusercontent.com/4514346/201531397-609bb7b6-3f5e-4e67-a9cb-64831d9de8f2.png)

@skandanc

**Comment by SkandanC** [comment]
Hi @joamatab, I can look into it. Could you tell me how you got it installed on Linux? I've been trying to get the Salt package installed on my WSL, but everytime I try to import the PCells, I run into a numpy error, see: https://github.com/gdsfactory/gdsfactory/issues/800#issuecomment-1312556938

**Comment by SkandanC** [comment]
![image](https://user-images.githubusercontent.com/100642027/202871393-8e10f02f-9fda-40dc-b1ca-cd93a6441c4b.png)
I got it working a bit. Here is a 'C' component for demonstration.

As of now, I can import the cells, but the pcell parameters don't seem to vary the geometry, but it must be a simple bug in my code.

**Comment by joamatab** [comment]
Awesome Skandan,

can you send a PR for this?

**Comment by SkandanC** [comment]
Yes, once I fix the bug, I will open 2 PR's: one here and one in gdsfactory. I had to rewrite the pcell function a bit.

**Comment by joamatab** [comment]
sounds good

when changing the pcell parameters you need to change the name to see any changes

it would be nice to compute the name

**Comment by SkandanC** [comment]
I managed to fix that bug with my new implementation too. Changing the paraemetrs changes the geometry in real time, but it does not change the name in my case. But I can fix that too.

**Comment by joamatab** [comment]
How about adding all this functionality into gdsfactory without the flayout dependency?
